packages/storybook-blocks/package.json

```diff
@@ -41,7 +41,8 @@
     "test": "jest --passWithNoTests",
     "test:update": "yarn test -u",
     "storybook": "storybook dev -p 7012",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "postinstall": "yarn build"
```

Comment:
When first installing the monorepo this will build the package after running `yarn install` to be able to run storybook without needing to build it manually